### PR TITLE
Display

### PR DESCRIPTION
**Pull request by GuillaumeGomez**

### CODE REVIEW

**Comment by gkoz** [comment]
Well, if we're going to print something it makes sense to make that and the overwriting of the old file conditional on actually having altered the contents. If nothing was changed it shouldn't say anything.

**Comment by GuillaumeGomez** [comment]
Hum... Right.

**Comment by GuillaumeGomez** [comment]
Updated.

File: cargo-crates-io-deps-only.py [hunk 1]
```diff
@@ -17,15 +17,23 @@ def manifests():
...
 	manifest = open(os.path.join(path, CARGO_TOML), 'r')
 	with open(os.path.join(path, CARGO_TOML_NEW), 'w') as new_manifest:
+		print_something = False
 		for line in manifest:
 			if re_git_or_path.match(line):
+				print_something = True
 				continue
 			new_manifest.write(line)
+		if print_something:
+			print("Updating '%s'" % os.path.join(path, CARGO_TOML_NEW))
```
Comment (gkoz) [inline cargo-crates-io-deps-only.py]: You probably want `CARGO_TOML` here.

File: cargo-crates-io-deps-only.py [hunk 2]
```diff
@@ -17,15 +17,23 @@ def manifests():
...
+			has_printed_something = True
 	manifest.close()
 	os.remove(os.path.join(path, CARGO_TOML))
```
Comment (gkoz) [inline cargo-crates-io-deps-only.py]: It'd be nice to remove `CARGO_TOML_NEW` and skip the renaming if there were no changes.

Comment (GuillaumeGomez) [inline cargo-crates-io-deps-only.py]: Sure.

File: cargo-crates-io-deps-only.py (hunk 1 above)
Comment (GuillaumeGomez) [inline cargo-crates-io-deps-only.py]: Indeed!

**Comment by GuillaumeGomez** [comment]
Updated.

**Comment by gkoz** [comment]
Thanks!

**Pull request merged**